saleor/checkout/utils.py

```diff
@@ -919,7 +946,9 @@ def is_valid_shipping_method(checkout, discounts):
     return True
 
 
-def get_shipping_price_estimate(checkout: Checkout, discounts, country_code):
+def get_shipping_price_estimate(
+    checkout: Checkout, discounts: "DiscountsListType", country_code: str
+):
```

Comment:
It returns `Money | TaxedMoney` I think? We could hint it 😄

Reply:
AFAIK `Optional[TaxedMoneyRange]`